Vue-Basic-Video-Chat/js/app.js

```diff
@@ -0,0 +1,105 @@
+Vue.component('publisher', {
+  template: '<div></div>',
+  props: {
+    session: {
+      type: OT.Session,
+      required: false
+    },
+    opts: {
+      type: Object,
+      required: false
+    }
+  },
+  mounted: function() {
+    publisher = OT.initPublisher(this.$el, this.opts, (err) => {
```

Comment:
Should this variable be declared?

Reply:
Yes, good catch. I also added an error handler for session.publish